### DISCUSSION TITLE
Chromium systemWebview is not up to date on f droid

### DISCUSSION THREAD

**Discussion by pepekinho** [bromite/bromite#2088] [General]
on f droid the bromite webview is up to date but the chromium webview isn't
 I don't know if this is a bug but it seems to be abandoned on f droid
tested on lineage os 13, 14 and 18.1
![combine_images](https://user-images.githubusercontent.com/87846076/170846974-403bf1db-220e-4bcf-a2cf-afa3158479ce.jpg)

**Comment by csagan5** [comment]
I can see all versions including latest `102.0.5005.67`

**Reply by ghost** [reply]
<details>
  <summary>I have the same (old version):</summary>
<img src="https://user-images.githubusercontent.com/106708194/171612811-2093f30f-4a66-4410-baff-bd67d7caec0d.png" style="width: 360px;">
<img src="https://user-images.githubusercontent.com/106708194/171612829-922b4ea5-9383-412a-99b0-60159170d8b6.png" alt="Versions">
</details>
The firmware of the device is native, certified.
Since the purchase, the device has not been flashed, no changes have been made (no root rights, no tvrp, no patches).

**Comment by badwater** [comment]
I've the same issue with Chromium SystemWebView from Bromite F-Droid repository:
<details>
  <summary>Aurora Droid 1.0.8 Screenshot</summary>
<img src="https://user-images.githubusercontent.com/2357219/171547047-39f4fdc4-2d71-4eff-b5ce-fb4595bf416f.jpg" />
</details>
<details>
  <summary>F-Droid 1.15.2 Screenshot</summary>
<img src="https://user-images.githubusercontent.com/2357219/171547053-a2523dfc-6781-48b6-b0c0-f6315f7c7bb8.jpg" />
</details>
<details>
  <summary>F-Droid Classic 1.2 Screenshot</summary>
<img src="https://user-images.githubusercontent.com/2357219/171547054-64351195-3f21-4330-8cab-34f498f90a1a.jpg" />
</details>

add.
<details>
  <summary>fdroidcl show com.android.webview</summary>
Package          : com.android.webview
Name             : Chromium SystemWebView
Summary          : Vanilla Chromium SystemWebView built side-by-side with Bromite; requires root for installation
Added            : 2018-07-14
Last Updated     : 2018-07-14
Version          :  (1652918540)
License          : BSD-3-Clause
Categories       : Internet
Website          : https://www.bromite.org/chromium
Source Code      : https://chromium.googlesource.com/chromium/src
Issue Tracker    : https://bugs.chromium.org/p/chromium/issues/list
Changelog        : https://chromium.googlesource.com/chromium/src/+log/102.0.5005.67
Donate           : https://www.bromite.org/#donate
Bitcoin          : bitcoin:bc1qlx7h0lj9z88g2xfeuwsrtfs77sxuhrxf2t28sw

Description :


Available Versions :

    Version : 96.0.4664.183 (1643895340)
    Size    : 99666918
    MinSdk  : 0
    ABIs    : x86_64

    Version : 96.0.4664.183 (1643895330)
    Size    : 87440380
    MinSdk  : 0
    ABIs    : arm64-v8a

    Version : 96.0.4664.183 (1643895290)
    Size    : 53799669
    MinSdk  : 0
    ABIs    : x86

    Version : 96.0.4664.183 (1643895280)
    Size    : 47135494
    MinSdk  : 0
    ABIs    : armeabi-v7a

    Version : 96.0.4664.104 (1639174440)
    Size    : 99662823
    MinSdk  : 0
    ABIs    : x86_64

    Version : 96.0.4664.104 (1639174430)
    Size    : 87440382
    MinSdk  : 0
    ABIs    : arm64-v8a

    Version : 96.0.4664.104 (1639174390)
    Size    : 53799667
    MinSdk  : 0
    ABIs    : x86

    Version : 96.0.4664.104 (1639174380)
    Size    : 47135494
    MinSdk  : 0
    ABIs    : armeabi-v7a

    Version : 96.0.4664.54 (1637110950)
    Size    : 99630046
    MinSdk  : 0
    ABIs    : x86_64

    Version : 96.0.4664.54 (1637110940)
    Size    : 87415797
    MinSdk  : 0
    ABIs    : arm64-v8a

    Version : 96.0.4664.54 (1637110900)
    Size    : 53770989
    MinSdk  : 0
    ABIs    : x86

    Version : 96.0.4664.54 (1637110890)
    Size    : 47110908
    MinSdk  : 0
    ABIs    : armeabi-v7a

    Version : 95.0.4638.79 (1637054290)
    Size    : 99213188
    MinSdk  : 0
    ABIs    : x86_64

    Version : 95.0.4638.79 (1637054280)
    Size    : 87056279
    MinSdk  : 0
    ABIs    : arm64-v8a

    Version : 95.0.4638.79 (1637054240)
    Size    : 53542542
    MinSdk  : 0
    ABIs    : x86

    Version : 95.0.4638.79 (1637054230)
    Size    : 46972579
    MinSdk  : 0
    ABIs    : armeabi-v7a

    Version : 95.0.4638.78 (1636070540)
    Size    : 99319687
    MinSdk  : 0
    ABIs    : x86_64

    Version : 95.0.4638.78 (1636070530)
    Size    : 87142292
    MinSdk  : 0
    ABIs    : arm64-v8a

    Version : 95.0.4638.78 (1636070490)
    Size    : 53595792
    MinSdk  : 0
    ABIs    : x86

    Version : 95.0.4638.78 (1636070480)
    Size    : 47021730
    MinSdk  : 0
    ABIs    : armeabi-v7a

</details>

Oh, this it is Bromite SystemWebView with package name com.android.webview versions 95-96...

**Comment by csagan5** [comment]
> Oh, this it is Bromite SystemWebView with package name com.android.webview versions 95-96...

Yes, now I see the problem; will try to fix this for next release.

**Reply by ghost** [reply]
Yeah, now the latest version is displayed correctly.

**Reply by badwater** [reply]
Thank you very much. The problem is solved.
